x-pack/plugins/observability_solution/infra/server/routes/custom_dashboards/save_custom_dashboard.ts

```diff
@@ -37,13 +39,17 @@ export function initSaveCustomDashboardRoute(framework: KibanaFramework) {
 
       await checkCustomDashboardsEnabled(uiSettingsClient);
 
-      const payload: InfraSaveCustomDashboardsRequestPayload = request.body;
-      const customDashboards = await findCustomDashboard(payload.assetType, savedObjectsClient);
+      const payload: InfraSaveCustomDashboardsRequestPayload = {
+        ...request.body,
+      };
```

Comment:
nit: request.body is already typed as InfraSaveCustomDashboardsRequestPayload

```suggestion
      const { id, ...payload } = request.body;
```

Reply:
the `id` is optional so I removed the type 👍